client_dashboard/common/src/commonMain/kotlin/org/thechance/common/di/UseCaseModule.kt

```diff
@@ -21,4 +12,5 @@ val UseCaseModule = module {
     singleOf(::CreateNewTaxiUseCase) { bind<ICreateNewTaxiUseCase>() }
     singleOf(::FindTaxiByUserNameUseCase) { bind<IFindTaxiByUsernameUseCase>() }
     singleOf(::GetRestaurantsUseCase) { bind<IGetRestaurantsUseCase>() }
-}
+    singleOf(::LoginUserUseCase) { bind<ILoginUserUseCase>() }
```

Comment:
what is the defferent between that and this way?
 single<ILoginUserUseCase> { LoginUserUseCase(get()) }

Reply:
Bareq suggested that

Reply:
cause its more clear